src/index.ts

```diff
@@ -268,6 +268,18 @@ class ClientApp {
      * @since 1.0.0
      */
     get pcEnvironment() {
+        return this.gcEnvironment;
+    }
+    /**
+     * Returns the gcEnvironment (e.g. mypurecloud.com, mypurecloud.jp) if known; null otherwise.
+     * This value will be available if a valid Genesys Cloud Environment is provided, inferred, or
+     * defaulted from the config passed to this instance.
+     *
+     * @returns the valid Genesys Cloud environment; null if unknown.
+     *
+     * @since 2.6.3
+     */
+    get gcEnvironment() {
```

Comment:
Once we can deprecate pcEnvironment, I think we'll also deprecate the string version of this in favor of targetEnvironment property of type `Environment` or `RuntimeEnvironment`.  The string verison is a bit ambiguous; but, for now, we can't reliably generate that type from the pcEnvironment input, so, we'll leave that as a breaking change for v3 of this lib or for appsv2